store/json.go

```diff
@@ -83,7 +89,12 @@ func (kvs *jsonFileStore) Read(key string, value interface{}) error {
 		}
 
 		if len(b) == 0 {
-			log.Printf("Unable to read file %s, was empty", kvs.fileName)
+			if kvs.logger != nil {
+				logger.Info("Unable to read empty file", zap.String("fileName", kvs.fileName))
```

Comment:
This would become, `kvs.logger.Info(..)`. And we remove the variable `logger` from line 23.
This is valid for all log in the file.

Reply:
fixed